main/src/cgeo/geocaching/maps/PositionHistory.java

```diff
@@ -21,6 +25,26 @@
 
     private ArrayList<Location> history = new ArrayList<>();
 
+    // load data from permanent storage
+    public PositionHistory() {
```

Comment:
PositionHistory is only used on CGeoMap if I am not mistaken, so the persistence will only work there, or did I miss something?